ako-gateway-api/lib/lib.go

```diff
@@ -1 +1,35 @@
+/*
+ * Copyright 2023-2024 VMware, Inc.
+ * All Rights Reserved.
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*   http://www.apache.org/licenses/LICENSE-2.0
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
 package lib
+
+import (
+	"github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pkg/utils"
+	"k8s.io/client-go/kubernetes"
+)
+
+func InformersToRegister(kclient *kubernetes.Clientset) ([]string, error) {
+	// Initialize the following informers in all AKO deployments. Provide AKO the ability to watch over
+	// Services, Endpoints, Secrets, ConfigMaps and Namespaces.
+	allInformers := []string{
+		utils.ServiceInformer,
+		utils.EndpointInformer,
+		utils.SecretInformer,
+		utils.ConfigMapInformer,
+		utils.NSInformer,
+		utils.NodeInformer,
```

Comment:
NodeInformer might not be required since we are not dealing with `vrf`.

Reply:
removed